network/src/connectivity_manager/mod.rs

```diff
@@ -52,9 +52,9 @@ use std::{
     collections::{HashMap, HashSet},
     fmt, mem,
     sync::{Arc, RwLock},
-    time::{Duration, Instant},
+    time::Duration,
 };
-use tokio::time;
+use tokio::{time, time::Instant};
```

Comment:
Is there a difference between these Instant's?

Reply:
Looking at the documentation, not really.  But, we only use this for logging time until the delay, so I figured I'd clean it up rather than convert types.